python/src/scipp/compat/mantid.py

```diff
@@ -192,8 +192,8 @@ def convert_Workspace2D_to_dataset(ws):
     return array
 
 
-def convert_EventWorkspace_to_dataset(ws, load_pulse_times, EventType):
-
+def convert_EventWorkspace_to_dataset(ws, load_pulse_times):
+    from mantid.api import EventType
```

Comment:
Is there a good reason to do lazy importing?

Reply:
Yes, we don't attempt to import anything Mantid related until `load()` is called.